charts/datadog/templates/cluster-agent-deployment.yaml

```diff
@@ -183,6 +183,10 @@ spec:
           - name: DD_ADMISSION_CONTROLLER_INJECT_CONFIG_LOCAL_SERVICE_NAME
             value: {{ template "localService.name" . }}
           {{- end }}
+          {{- if .Values.providers.aks.enabled }}
+          - name: DD_ADMISSION_CONTROLLER_ADD_AKS_SELECTORS
+            value: "true"
+          {{- end }}
           {{- end }}
```

Comment:
@vboulineau 

This new check for `.Values.providers.aks.enabled` appears to be incorrectly nested inside of the check for `.Values.clusterAgent.admissionController.configMode` instead of under `.Values.clusterAgent.admissionController.enabled`